test/e2e_node/container_lifecycle_test.go

```diff
@@ -3317,6 +3330,12 @@ var _ = SIGDescribe(nodefeature.SidecarContainers, "Containers Lifecycle", func(
 		pod, err = client.Get(context.TODO(), pod.Name, metav1.GetOptions{})
 		framework.ExpectNoError(err)
 
+		for _, status := range pod.Status.InitContainerStatuses {
```

Comment:
This code is repeated many times, can it be replaced by a function?

Reply:
Thanks! I reworked it and added a helper function to abstract the testing logic.